pkg/operator/ceph/csi/csi.go

```diff
@@ -40,10 +41,22 @@ func ValidateAndConfigureDrivers(context *clusterd.Context, namespace, rookImage
 	}
 
 	if CSIEnabled() {
-		if err := startDrivers(context.Clientset, context.RookClientset, namespace, serverVersion, ownerInfo, v); err != nil {
-			logger.Errorf("failed to start Ceph csi drivers. %v", err)
-			return
+		csiLock.Lock()
+		for i := maxRetries - 1; i >= 0; i-- {
```

Comment:
Let's just define it directly here for code readability. I'd also suggest counting up as traditional for readability
```suggestion
		maxRetries := 3	
		for i := 0; i < maxRetries; i-- {
```